.github/workflows/publish-beta-release.yml

```diff
@@ -0,0 +1,31 @@
+name: Manual NPM Publish (Beta Release)
```

Comment:
For now lets not change this.  I would like to do some more investigation into streamlined git workflows.